src/Device/Driver.hpp

```diff
@@ -101,6 +101,24 @@ class Device {
   virtual bool PutBallast(double fraction, double overload,
                           OperationEnvironment &env) = 0;
 
+  /**
+    * Send the new crew weight value to the device.
+    *
+    * @param total weight in kg of all crew members
```

Comment:
kilogram is not a weight unit, but a mass unit. The unit of weight would be Newton.

Reply:
weight needs to be replaced by mass in PR for plane masses